extensions/extensions-docker/aissemble-spark-operator/src/main/resources/docker/Dockerfile

```diff
@@ -16,40 +16,20 @@
 ARG DOCKER_BASELINE_REPO_ID
 ARG VERSION_AISSEMBLE
 
-FROM golang:1.22.2-alpine as builder
-
-WORKDIR /workspace
-
-# Copy the Go Modules manifests
-COPY ./target/checkout/go* ./
-# Cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
-
-# Copy the go source code
-COPY ./target/checkout/main.go main.go
-COPY ./target/checkout/pkg/ pkg/
-COPY ./target/checkout/hack/gencerts.sh /tmp/scripts/gencerts.sh
-COPY ./target/checkout/entrypoint.sh /tmp/scripts/entrypoint.sh
-
-RUN go mod tidy
-# Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o /usr/bin/spark-operator main.go
+FROM kubeflow/spark-operator:v1beta2-1.6.2-3.5.0 AS builder
 
+# We would be able to use the kubeflow image directly, except that it is on Spark 3.5 instead of 3.4
 FROM ${DOCKER_BASELINE_REPO_ID}boozallen/aissemble-spark:${VERSION_AISSEMBLE}
 
 LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
 
 USER root
 
-COPY --from=builder /usr/bin/spark-operator /usr/bin/
-COPY --from=builder /tmp/scripts/* /usr/bin/
-RUN apt-get update --allow-releaseinfo-change \
-    && apt-get update \
-    && apt-get install -y openssl curl tini \
-    && rm -rf /var/lib/apt/lists/* \
-    && chmod +x /usr/bin/entrypoint.sh \
-    && chmod +x /usr/bin/gencerts.sh
+COPY --from=builder --chmod=755 /usr/bin/spark-operator /usr/bin/
+COPY --from=builder --chmod=755 /usr/bin/entrypoint.sh /usr/bin/
+RUN apt-get update \
+    && apt-get install -y tini \
```

Comment:
I: Spark Operator 1.6+ no longer uses the gencerts script or the webhook init/cleanup k8s jobs.  So we only need `tini` as it's used in `entrypoint.sh` to launch the service.